docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-hints-autodiscover.asciidoc

```diff
@@ -134,6 +133,10 @@ initContainers:
       mountPath: /etc/elastic-agent/inputs.d
 ----
 
+
+NOTE: The {agent} can read from `{path.config}/inputs.d`  multiple configuration files and finally produce a unified one (refer to <<elastic-agent-configuration>>). Users have the ability to manually mount their own templates under `/etc/elastic-agent/inputs.d` if they want to skip enabling initContainers section.
```

Comment:
```suggestion
NOTE: The {agent} can load multiple configuration files from `{path.config}/inputs.d`  and finally produce a unified one (refer to <<elastic-agent-configuration>>). Users have the ability to manually mount their own templates under `/etc/elastic-agent/inputs.d` if they want to skip enabling initContainers section.
```

Is `<<elastic-agent-configuration>>` the link to the specific section that refers to the `inputs.d` "feature"? If we can point to this specific part it would really handy.

Reply:
This is the link of elastic-agent-configuration.asciidoc file

Reply:
I was mostly thinking of linking directly to docs  page part that refers the `inputs.d` usage. But I see that `inputs.d` is only mentioned in https://www.elastic.co/guide/en/fleet/current/elastic-agent-configuration.html and under no specific tittle/section so it's not possible to link directly to its section. Anyways we can add it as is and improve it further in the future is there are asks for it.